pkg/model/serializer.go

```diff
@@ -47,6 +47,38 @@ func (e EnvVar) MarshalYAML() (interface{}, error) {
 	return e.Name + "=" + e.Value, nil
 }
 
+// UnmarshalYAML Implements the Unmarshaler interface of the yaml pkg.
+func (c *Command) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var multi []string
+	err := unmarshal(&multi)
+	if err != nil {
+		var single string
+		err := unmarshal(&single)
+		if err != nil {
+			return err
+		}
+		if strings.Contains(single, " ") {
+			c.Values = []string{"sh", "-c", single}
+		} else {
+			c.Values = []string{single}
+		}
+	} else {
+		c.Values = multi
+	}
+	return nil
+}
+
+// MarshalYAML Implements the marshaler interface of the yaml pkg.
+func (c Command) MarshalYAML() (interface{}, error) {
+	if len(c.Values) == 1 && !strings.Contains(c.Values[0], " ") {
+		return c.Values[0], nil
+	}
+	if len(c.Values) == 3 && c.Values[0] == "sh" && c.Values[1] == "-c" && !strings.Contains(c.Values[2], " ") {
```

Comment:
Having the `sh -c` on the command is odd. I think that's something we should add during the execution, but not the manifest.

Reply:
It is the other way around. It is to remove them when marshaling. Although thinking twice, I don't need this even for the `init` command